src/tests/api/auth/sign-in/validate.spec.ts

```
@@ -0,0 +1,285 @@
+			result = err;
+		}
+
+		expect(result).toBeInstanceOf(CustomError);
+		expect(result.statusCode).toBe(StatusCodeEnum.BAD_REQUEST);
+		expect(result.message).toBe(
+			'password must match the following: "/^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@#$!%*?&])[A-Za-z\\d@$!%*#?&]{6,100}$/"',
+		);
+	});
+
+	it("should throw error with invalid password (less than 6 characters)", async () => {
+		let result: any;
+
+		try {
+			result = await validate({
+				email: validEmail,
+				password: "aA@12",
+			});
+		} catch (err: any) {
+			result = err;
+		}
+
+		expect(result).toBeInstanceOf(CustomError);
+		expect(result.statusCode).toBe(StatusCodeEnum.BAD_REQUEST);
+		expect(result.message).toBe(
+			'password must match the following: "/^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@#$!%*?&])[A-Za-z\\d@$!%*#?&]{6,100}$/"',
+		);
+	});
+
+	it("should throw error with invalid password (don't have a number)", async () => {
+		let result: any;
+
+		try {
+			result = await validate({
+				email: validEmail,
+				password: "JohnLiked@",
+			});
+		} catch (err: any) {
+			result = err;
+		}
+
+		expect(result).toBeInstanceOf(CustomError);
+		expect(result.statusCode).toBe(StatusCodeEnum.BAD_REQUEST);
+		expect(result.message).toBe(
+			'password must match the following: "/^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@#$!%*?&])[A-Za-z\\d@$!%*#?&]{6,100}$/"',
+		);
+	});
+
+	it("should throw error with invalid password (don't have a symbol)", async () => {
+		let result: any;
+
+		try {
+			result = await validate({
+				email: validEmail,
+				password: "JohnLiked123",
+			});
+		} catch (err: any) {
+			result = err;
+		}
+
+		expect(result).toBeInstanceOf(CustomError);
+		expect(result.statusCode).toBe(StatusCodeEnum.BAD_REQUEST);
+		expect(result.message).toBe(
+			'password must match the following: "/^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@#$!%*?&])[A-Za-z\\d@$!%*#?&]{6,100}$/"',
+		);
+	});
+
+	it("should throw error with invalid email (don't have a at-sign)", async () => {
```

- This should be inside the `describe`.
- `it("should throw an error with invalid email"`